### PR TITLE
Fix libmpg123 not found

### PR DESCRIPTION
**Pull request by Yonle**
This fix includes a fix of missing library of mpg123 for first time installation

### CODE REVIEW

File: packages/chocolate-doom/build.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ TERMUX_PKG_VERSION=3.0.1
 TERMUX_PKG_REVISION=5
```
Comment (xeffyr) [inline packages/chocolate-doom/build.sh:6]: Increment revision when modifying package information.

**Review by xeffyr — CHANGES_REQUESTED** [review]

File: packages/chocolate-doom/build.sh (hunk 1 above)
Comment (Yonle) [reply packages/chocolate-doom/build.sh:6]: Changes allowed

**Review by Yonle — COMMENTED** [review]
Solving this....

File: packages/chocolate-doom/build.sh (hunk 1 above)
Comment (xeffyr) [reply packages/chocolate-doom/build.sh:6]: ?

Снимок экрана от 2021-07-06 16-07-56

**Comment by Yonle** [comment]
Closing due to Reforked Repo.

**Pull request closed without merging**